tket2/src/optimiser/badger.rs

```diff
@@ -141,15 +143,20 @@ where
         log_config: BadgerLogger,
         options: BadgerOptions,
     ) -> Circuit {
-        if options.split_circuit && options.n_threads.get() > 1 {
-            return self.split_run(circ, log_config, options).unwrap();
-        }
         match options.n_threads.get() {
             1 => self.badger(circ, log_config, options),
-            _ => self.badger_multithreaded(circ, log_config, options),
+            _ => {
+                if options.split_circuit {
+                    self.badger_split_multithreaded(circ, log_config, options)
```

Comment:
I'm a bit surprised there isn't coverage of this

Reply:
Agreed -- I've opened an issue about it.

Reply:
see #505